Improve docs around setting admin usernames in tap-values file (#84)
pivotal/docs-tas-adapter: troubleshooting.md

#### Possible causes

1. The user creating the CFOrg does not have a `RoleBinding` that is bound to the `korifi-controllers-admin` ClusterRole.

> 2. The user has a `RoleBinding` but the binding is assigned to a username that is defined in the user's local Kubeconfig but isn't a recognized user on the Kubernetes cluster. 

Reviewer: ```suggestion
2. The user appears to have a `RoleBinding`, but the binding is assigned to a local Kubeconfig user entry that does not match a recognized user on the Kubernetes cluster.
```

Reviewer: This one's hard to word. 😅 I suggested a tweak, but not sure myself if that is any clearer. Maybe @HenryBorys can help us out here. 🙂

Reviewer 2: Looks good. I just removed a couple of extra words.
